src/frontend/src/templates/template.js

```diff
@@ -37,7 +37,7 @@ const tutorialSteps = [
 
 const useStyles = makeStyles((theme) => ({
   root: {
-    flexGrow: 1,
+    // flexGrow: 1,
```

Comment:
Please remove this comment. Then I think we are good to go.

Reply:
Sorry, forget to remove it. I have fixed it. Thank you.